contracts/plugins/lido/README.md

```diff
@@ -0,0 +1,35 @@
+# Lido stETH Collateral Plugin
+
+## Summary
+
+This plugin allows `wstETH` holders use their tokens as collateral in the Reverse Protocol.
+
+As described in the [Lido Site](https://docs.lido.fi/guides/steth-integration-guide#wsteth) , `wstETH` is a LSD (Liquid staking derivatives) which enables users to sell or transfer stacked ETH even before withdrawal being enabled.
+
+`wstETH` will accrue revenue from **staking rewards** into itself by **increasing** the exchange rate of the `wstETH` per `stETH`.
```

Comment:
"...the exchange rate of `stETH` per `wstETH`"

Reply:
Fixed